src/django/api/models/submission.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.contrib.gis.db import models as gis_models
+
+from .boundary import Boundary
+from .role import Roles
+from .user import User
+
+__all__ = ["Submission", "Review", "Approval", "Annotation"]
+
+
+def limit_by_validator_or_admin():
+    return models.Q(role__pk=Roles.VALIDATOR) | models.Q(role__pk=Roles.ADMINISTRATOR)
```

Comment:
A possibility if you wish to avoid nesting `Q` objects.
```suggestion
    return models.Q(role__pk__in=[Roles.VALIDATOR, Roles.ADMINISTRATOR])
```